src/plots/barPlot.ts

```diff
@@ -397,6 +397,35 @@ export module Plots {
       }
     }
 
+    /**
+     * Makes sure the extent takes into account the widths of the bars
+     */
+    protected _extentsForProperty(property: string) {
+      let extents = super._extentsForProperty(property);
+
+      let accScaleBinding: Plots.AccessorScaleBinding<any, any>;
+      if (property === "x" && this._isVertical) {
+        accScaleBinding = this.x();
+      } else if (property === "y" && !this._isVertical) {
+        accScaleBinding = this.y();
+      } else {
+        return extents;
+      }
+
+      if (!(accScaleBinding.scale instanceof QuantitativeScale)) {
```

Comment:
Potential for error if `accScaleBinding` is `undefined`, which is possible if `x()` or `y()` hasn't been set up by the user.